### PR TITLE
Implementing Census Communal Establishment schema with selenium tests

### PR DESCRIPTION
**Pull request by jonnyshaw89**
### What is the context of this PR?
Implementing Census Communal Establishment schema with selenium tests

### How to review 
Load schema census_communal.json and confirm it matches slides.

### CODE REVIEW

File: app/data/census_communal.json [hunk 1]
```diff
@@ -10,46 +10,567 @@
...
+                    "title": "Establishment",
+                    "sections": [
+                        {
+                            "title": "Establishment",
+                            "id": "bed-spaces-section",
+                            "questions": [
+                                {
+                                    "id": "bed-spaces-question",
+                                    "title": "2 How many guest bed spaces does your establishment offer?",
+                                    "type": "General",
```
Comment (benjefferies) [inline app/data/census_communal.json:82]: Does this need guidance for "Count double rooms as 2 guest bed spaces"

File: app/data/census_communal.json [hunk 2]
```diff
@@ -10,46 +10,567 @@
...
+                    "title": "Completion Preferences",
+                    "sections": [
+                        {
+                            "title": "Completion Preferences",
+                            "id": "completion-preference-individual-section",
+                            "questions": [
+                                {
+                                    "id": "completion-preference-individual-question",
+                                    "title": "1 If tomorrow was Census Day, to the best of your knowledge, how do you think your usual residents would prefer to complete their individual questionnaires?",
+                                    "description": "For the 2021 Census all hotels, B&B, guest houses etc. will be expected to complete a questionnaire about their establishment. All usual residents will be required to complete an individual questionnaire (with more personal details).",
```
Comment (benjefferies) [inline app/data/census_communal.json]: Should this appear as guidance or something as it looks in the wrong order

**Review by benjefferies — CHANGES_REQUESTED** [review]

File: app/data/census_communal.json (hunk 1 above)
Comment (jonnyshaw89) [reply app/data/census_communal.json:82]: will add as description.

File: app/data/census_communal.json (hunk 2 above)
Comment (jonnyshaw89) [reply app/data/census_communal.json]: I don't think we actually have anything that displays in that order?

Comment (benjefferies) [reply app/data/census_communal.json]: This is the only thing that looked really wrong against the slides anyway

Comment (jonnyshaw89) [reply app/data/census_communal.json]: This has been resolved now with the section description

**Review by benjefferies — APPROVED** [review]

**Review by samiwel — APPROVED** [review]

**Review by ajmaddaford — APPROVED** [review]

**Pull request merged**